src/track/beatutils.cpp

```diff
@@ -206,18 +204,17 @@ double BeatUtils::calculateBpm(const QVector<double>& beats, int SampleRate,
      * non-electronic music isn't too bad.
      */
 
-    //qDebug() << "BPM range between " << min_bpm << " and " << max_bpm;
+    // qDebug() << "BPM range between " << min_bpm << " and " << max_bpm;
```

Comment:
Disabled code without a space.

Reply:
clang-format is not able to parse the text in comments and make a decision

Reply:
Commented out code should be deleted anyway.

Reply:
This is IMHO an expedition, since it allows to reuse the debug statement during later development.